### PR TITLE
Created a page for the New Relic Builder's

### PR DESCRIPTION
**Pull request by pachicodes**
## Description
The DevRel team is launching a program called The New Relic builders, and it needs a page on the developers website. This PR adds said page.

## Reviewer Notes

If there are links or steps needed to test this work, add them here.

## Related Issue(s) / Ticket(s)

If there are any related [GitHub Issues](https://github.com/newrelic/developer-website/issues) or JIRA tickets, add links to them here.
* [issue]()

## Screenshot(s)

If relevant, add screenshots here.

## Use Conventional Commits

Please help the maintainers by leveraging the following [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/)
standards in your pull request title and commit messages.

## Use `chore`

* for minor changes / additions / corrections to content.
* for minor changes / additions / corrections to images.
* for minor non-functional changes / additions to github actions, github templates, package or config updates, etc

```bash
git commit -m "chore: adjusting config and content"
```

## Use `fix`

* for minor functional corrections to code.

```bash
git commit -m "fix: typo and prop error in the code of conduct"
```

## Use `feat`

* for major functional changes or additions to code.

```bash
git commit -m "feat(media): creating a video landing page"
```

### CODE REVIEW

File: src/components/FeatherIcon.js [hunk 1]
```diff
@@ -145,7 +145,6 @@ const ICONS = {
       <rect x="14" y="7" width="3" height="5" />
     </>
   ),
-  github: <path d="M9 19c-5 1.5-5-2.5-7-3m14 6v-3.87a3.37 3.37 0 0 0-.94-2.61c3.14-.35 6.44-1.54 6.44-7A5.44 5.44 0 0 0 20 4.77 5.07 5.07 0 0 0 19.91 1S18.73.65 16 2.48a13.38 13.38 0 0 0-7 0C6.27.65 5.09 1 5.09 1A5.07 5.07 0 0 0 5 4.77a5.44 5.44 0 0 0-1.5 3.78c0 5.42 3.3 6.61 6.44 7A3.37 3.37 0 0 0 9 18.13V22" />,
```
Comment (zstix) [inline src/components/FeatherIcon.js:148]: Out of curiosity, was this intentionally removed?

File: src/pages/builders.js [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+      <SEO />
+      <PageLayout type={PageLayout.TYPE.SINGLE_COLUMN}>
+        <PageLayout.Header title="New Relic Builders" />
+        <PageLayout.Content>
+          <section className={cx(styles.section, styles.twoColumn)}>
+            <div>
+              <p>
+                {' '}
+                Our goal with the New Relic Community, also known as{' '}
+                <a href="https://www.therelicans.com/">The Relican's</a>, is to
```
Comment (zstix) [inline src/pages/builders.js:24]: It seems like we're styling this without a `'`.
```suggestion
                <a href="https://www.therelicans.com/">The Relicans</a>, is to
```

File: src/components/FeatherIcon.js (hunk 1 above)
Comment (pachicodes) [reply src/components/FeatherIcon.js:148]: Yes, my ESlint was showing 2 identical of those and I was getting an error, so I removed one

**Review by zstix — APPROVED** [review]
This looks fantastic 🎉

**Comment by jerelmiller** [comment]
@pachicodes It looks like you have a conflict. We recently did a few updates to that relicans page in #1073 to fix some of the eslint errors. It looks like maybe you're behind that change. Could you pull that in?

**Comment by pachicodes** [comment]
I pulled that, so it should be good now. 
Thank you!!!

**Review by jerelmiller — APPROVED** [review]
Thanks for updating those changes!

**Pull request merged**